filesystem/ext2.ksy

```diff
@@ -23,14 +23,15 @@ types:
       - id: super_block
         type: super_block_struct
         size: 1024
-      # http://www.nongnu.org/ext2-doc/ext2.html#BLOCK-GROUP-DESCRIPTOR-TABLE
+        doc-ref: http://www.nongnu.org/ext2-doc/ext2.html#BLOCK-GROUP-DESCRIPTOR-TABLE
       - id: block_groups
         type: bgd
         repeat: expr
         repeat-expr: super_block.block_group_count
-  # http://www.nongnu.org/ext2-doc/ext2.html#super_block
-  # http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/fs/ext2/ext2.h#n416
-  # https://ext4.wiki.kernel.org/index.php/Ext4_Disk_Layout#The_Super_Block
+        doc-ref:
+        - http://www.nongnu.org/ext2-doc/ext2.html#super_block
+        - http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/fs/ext2/ext2.h#n416
+        - https://ext4.wiki.kernel.org/index.php/Ext4_Disk_Layout#The_Super_Block
   super_block_struct:
```

Comment:
I strongly suspect that these links are not meant to be associated with the attribute `block_groups` but rather with the following type `super_block_struct`. See the original version of this spec:

https://github.com/kaitai-io/kaitai_struct_formats/blob/2eec205abad07f9151fa5097083e4e549d09673f/filesystem/ext2.ksy#L26-L37